src/views/HomeView.vue

```diff
@@ -1,39 +1,129 @@
-            />
+            <line y1="2" x2="127" y2="2" stroke="#E2B764" />
           </svg>
-        </template>
-        <template #text> Title </template>
-      </IdeaCard>
-    </main>
+          <LightButton>Learn More</LightButton>
+        </div>
+      </div>
+
+      <div
+        class="mt-20 flex flex-col gap-y-12 lg:flex-row justify-around items-center bg-dark opacity-90 p-10 lg:p-8"
+      >
+        <div v-for="index in 3" :key="index">
+          <div class="flex flex-col w-80 gap-4">
+            <div class="w-80 h-80 bg-white"></div>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+              exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
Not sure if I like how this appears on mobile. The items seem to just go straight down, which looks kinda uggo. Is it possible to scale them to make them fit on one row? Or maybe add a carousel?